bzzeth/bzzeth.go

```diff
@@ -118,7 +126,7 @@ func (b *BzzEth) handleMsg(p *Peer) func(context.Context, interface{}) error {
 func (b *BzzEth) handleMsgFromSwarmNode(p *Peer) func(context.Context, interface{}) error {
 	return func(ctx context.Context, msg interface{}) error {
 		p.logger.Warn("bzzeth.handleMsgFromSwarmNode")
-		return errRcvdMsgFromSwarmNode
+		return protocols.BreakError(errRcvdMsgFromSwarmNode)
```

Comment:
Warning should be removed, even more as it does not add any context to the error.